src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -314,7 +314,7 @@ function initAugmentations() {
     const WiredReflexes = new Augmentation({
         name:AugmentationNames.WiredReflexes, repCost:1.25e3, moneyCost:2.5e6,
         info:"Synthetic nerve-enhancements are injected into all major parts of the somatic nervous system, " +
-             "supercharging the body's ability to send signals through neurons. This results in increased reflex speed.<br><br>" +
+             "supercharging the spread of neural signals, increasing reflex speed.<br><br>" +
```

Comment:
Can we change this to "supercharging the spread of neural signals and increasing reflex speed"